src/analyseReport.js

```diff
@@ -0,0 +1,106 @@
+const fs = require("fs");
+
+let specs = [];
+const browsers = ["chrome", "firefox"];
+const defaultBrowser = "chrome"; // I will use it in case of there is no browser in the title.
+
+function getBrowserFromTitle(title) {
+  title = title.toLowerCase();
+  for (let browser of browsers) {
+    if (title.includes(browser)) {
+      return browser;
+    }
+  }
+}
+
+function orderBasedOnBrowserDuration(browser) {
+  let aDuration, bDuration;
+  return specs.sort(function (a, b) {
+    for (let i = 0; i < a.data.length; i++) {
+      if (a.data[i].browser === browser) {
+        aDuration = a.data[i].duration;
+        break;
+      }
+    }
```

Comment:
@0xIslamTaha What is the purpose of `for` loops here? 🤔 
`aDuration` and `bDuration` will always equal the last item in the array after each loop run.
Did you want to sum the durations from specs? 🤔

Reply:
@RadekM86 
I need to return the duration based on the browser value. `a.data = [ {browser: 'chrome', duration: 1}, {browser: 'firefox', duration: 2} ]`. I updated the implementation using the filter method.